.travis.yml

```diff
@@ -21,6 +21,7 @@ script:
   # Guard against invalid escapes in strings, like '\s'.
   - python -We:invalid -m compileall -f mpmath -q
   - pycodestyle
+  - flake8 . --count --select=E9,F63,F7 --show-source --statistics
```

Comment:
1) You might put most flake8 args to setup.cfg
2) run pycodestyle on the line above will be redundant, I think?